tests/API/Integration/Scaffolding/DatabaseContainer.cs

```diff
@@ -98,6 +98,14 @@ RESTART IDENTITY
             // });
             
             peopleContext.SaveChanges();
+
+            // Account for the identities of Units we've already added to the database.  Prevents error:
+            //      duplicate key value violates unique constraint
+            peopleContext.Database.ExecuteSqlRaw(@"
+                SELECT
+                    setval(pg_get_serial_sequence('public.units', 'id'), 
+                    (SELECT MAX(id) FROM public.units));
+                ");
```

Comment:
Just for my own curiosity, what prompted this?

Reply:
Basically it's how Postgres does what SQL Server does with `IDENTITY_INSERT`.

When you tried to create a new Unit you would get a primary key conflict on `Id: 1`
Basically our two units we insert from `TestEntities.Units` get inserted with Id's already set, so the sequence that generates `Unit.Id` still thinks the next Unit inserted should get `1`.

This updates the sequence that the next Unit created should have Id `3`.

Reply:
Perfect. Good catch.